test/langtools/tools/javac/valhalla/lworld-values/ProjectionRelationsTest.java

```diff
@@ -0,0 +1,279 @@
+/*
```

Comment:
You might want to look at this test framework:

test/langtools/tools/lib/types/TypeHarness.java

An example of such a test is:

test/langtools/tools/javac/types/PrimitiveConversionTest.java

Reply:
Thanks. This could come in handy for future. Appreciated.